client/my-sites/site-settings/jetpack-credentials/index.jsx

```diff
@@ -25,13 +26,28 @@ import RewindCredentialsForm from 'components/rewind-credentials-form';
 import './style.scss';
 
 class JetpackCredentials extends Component {
+	isSectionHighlighted() {
+		if ( ! window.location.hash ) {
+			return null;
+		}
+
+		const hash = window.location.hash.substring( 1 );
+		if ( 'credentials' === hash ) {
+			return true;
+		}
```

Comment:
lets always return a false here as well. 
```suggestion
		}
		return false;
```